src/HW7.java

```diff
@@ -0,0 +1,25 @@
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class HW7 {
+
+    public static void main(String[] args) {
+
+    List <String> phone = new ArrayList<>();
+    phone.add("+380963345434");
+    phone.add("80963345434");
+    phone.add("+480963345434");
+    phone.add("+380963345");
+
+    String regex = "^((8|\\+3)[\\- ]?)?(\\(?\\d{3,4}\\)?[\\- ]?)?[\\d\\- ]{5,10}$";
```

Comment:
Please format your code

Reply:
Where is the task 1?

Reply:
public static void divideArrayElements(int[] array, int divisor) {
        int[] dividedArray = new int[array.length];

        try {
            for (int i = 0; i < array.length; i++) {
                dividedArray[i] = array[i] / divisor;
            }
            System.out.println("Divided array elements:");
            for (int num : dividedArray) {
                System.out.println(num);
            }
        } catch (ArithmeticException e) {
            System.out.println("Error: Division by zero is not allowed.");
        } finally {
            System.out.println("Finally block executed.");
        }
    }